test/test_ops.py

```diff
@@ -199,28 +199,31 @@ def wrapped(*args):
 def get_autograd_fn_and_args_with_cotangents(f, sample, cotangents):
     args = tuple([sample.input] + list(sample.args))
     kwargs = sample.kwargs
-    flat_args, args_spec = tree_flatten(args)
+    flat_args, _ = tree_flatten(args)
     flat_cotangents, cotangents_spec = tree_flatten(cotangents)
 
     @functools.wraps(f)
-    def wrapped(*args):
-        assert len(args) == len(flat_args) + len(flat_cotangents)
-        actual_args = args[:len(flat_args)]
-        cotangents = args[len(flat_args):]
-        actual_args = tree_unflatten(actual_args, args_spec)
-        cotangents = tree_unflatten(cotangents, cotangents_spec)
+    def wrapped(*cotangents):
```

Comment:
The comment above `get_autograd_fn_and_args_with_cotangents` needs to be updated, right? This returns a new function that accepts cotangents and returns the vjp

Reply:
I'd also change the name of this function to reflect that it's returning something that only accepts cotangents